src/lib/y2network/interface.rb

```diff
@@ -18,6 +18,13 @@
 # find current contact information at www.suse.com.
 module Y2Network
   # Network interface.
+  #
+  # name and hardware attributes should be imutable for the whole life of the object.
```

Comment:
If it's immutable, then change to `attr_reader :name` or explain what prevents it.

What hardware attributes are we talking about? There are none here.

BTW, i*mm*utable

Reply:
it has leaked from https://github.com/yast/yast-network/pull/796, I'll remove the comment